package.json

```diff
@@ -123,8 +123,8 @@
     "test": "npm run test:unit && npm run test:integration && npm run test:e2e && npm run test:compat",
     "test:compat": "ava test/compat/**/*.test.js",
     "test:compat-all": "nve 14.18.0,16.13.0,18.0.0,19.0.0,20.0.0 npm run test:compat --ignore-scripts",
-    "test:e2e": "ava test/e2e/**/*.test.js --timeout 1m",
-    "test:integration": "ava test/integration/**/*.test.js --timeout 1m",
+    "test:e2e": "ava test/e2e/**/*.test.js --timeout 5m",
+    "test:integration": "ava test/integration/**/*.test.js --timeout 5m",
```

Comment:
This is not an ideal long-term solution, but I'd consider it fine for now. See also #1142.

Reply:
Resolved with ef535bbfcf7eef84343c9e37e92ecf4029a2de6d